.github/workflows/pythonpackage.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       uses: ricardochaves/python-lint@v1.1.0
       with:
         python-root-list: pyvistaqt
-        use-pylint: false
+        use-pylint: true
```

Comment:
```suggestion
        use-pylint: true
        extra-pylint-options: --disable=F0401
```

Reply:
I don't think it is necessary to check the import error in pylint because it can be checked in the test.

Reply:
> I don't think it is necessary to check the import error in pylint because it can be checked in the test.

I agree. Go for it @tkoyama010 :+1: